app/app/test/page.jsx

```diff
@@ -1,87 +1,92 @@
-    setUserAnswers(updatedAnswers);
-    if (currentQuestionIndex < questions.length - 1) {
-      setCurrentQuestionIndex(currentQuestionIndex + 1);
-    }
-  };
-
-  const handleNext = () => {
-    if (currentQuestionIndex < questions.length - 1) {
-      setCurrentQuestionIndex(currentQuestionIndex + 1);
-    }
-  };
-
-  const handlePrevious = () => {
-    if (currentQuestionIndex > 0) {
-      setCurrentQuestionIndex(currentQuestionIndex - 1);
-    }
+  const handleBeginExam = () => {
+    router.push('/questions');
   };
 
-  if (questions.length === 0) {
-    return <div>Loading...</div>;
-  }
-
-  const currentQuestion = questions[currentQuestionIndex];
-  const answers = currentQuestion.answers;
-
   return (
-    <div className="flex min-h-screen items-center justify-center bg-white">
-      <div className="card mx-auto w-full max-w-4xl rounded-xl bg-white p-10">
-        <QuestionCard
-          question={currentQuestion.question}
-          answers={answers}
-          onAnswer={handleAnswer}
-        />
-
-        <div>
-          <PreviousNext
-            showPrevious={currentQuestionIndex > 0}
-            showNext={currentQuestionIndex < questions.length - 0}
-            onPrevious={handlePrevious}
-            onNext={handleNext}
+    <div className="flex min-h-screen flex-col items-center justify-center bg-gray-100 p-4">
+      <div className="flex w-full max-w-4xl flex-col items-center justify-center rounded-lg bg-gray-100 p-8">
+        <div className="mb-6 flex w-full justify-center">
+          <img
+            src="https://c8.alamy.com/comp/2YXN0R4/edtech-concept-student-undertaking-an-online-test-with-a-ticking-clock-signaling-limited-time-efficient-assessment-in-digital-education-vector-illustration-2YXN0R4.jpg"
+            alt="Exam Instructions"
+            className="max-h-60 w-1/2 rounded-lg object-cover"
           />
         </div>
-        <div className="page-counter mt-5 text-center text-sm text-gray-600">
-          <strong>
-            {currentQuestionIndex + 1} / {questions.length}
-          </strong>
+
+        <h1 className="mb-6 text-center text-3xl font-bold text-blue-600">
+          Velkommen til din eksamen
+        </h1>
+
+        <div className="mb-6 grid grid-cols-1 gap-6 text-lg text-gray-700 md:grid-cols-2">
+          <div className="flex items-start gap-4">
+            <FontAwesomeIcon
+              icon={faArrowsRotate}
+              className="size-6 text-blue-500"
+            />
+            <span>
+              <strong>Undgå at opdatere eller genindlæse:</strong> Hvis du
+              opdaterer eller genindlæser eksamenssiden, vil du miste alt dit
+              fremskridt. Undgå at gøre dette under eksamen.
+            </span>
+          </div>
+          <div className="flex items-start gap-4">
+            <FontAwesomeIcon icon={faClock} className="size-6 text-green-500" />
+            <span>
+              <strong>Timeren starter:</strong> Eksamens timeren begynder, så
```

Comment:
I can see you're using tag `strong` a lot, are you sure it is necessary in all cases? 

<img width="1153" alt="image" src="https://github.com/user-attachments/assets/cc80f47f-201b-4008-8f64-e7d17980d4fc" />

https://devdocs.io/html/element/strong